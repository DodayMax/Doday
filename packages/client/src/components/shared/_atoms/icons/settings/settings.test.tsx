import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@shared';

describe('Settings', () => {
  it('renders correctly without props', () => {
    const settings = shallow(<Icons.Settings />);

    expect(settings).toMatchSnapshot();
  });
});
