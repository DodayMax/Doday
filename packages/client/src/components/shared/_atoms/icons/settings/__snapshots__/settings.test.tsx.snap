// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings renders correctly without props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <g
        fill="black"
        fillRule="evenodd"
      >
        <path
          d="M12.108 8.04a3.911 3.911 0 1 0 0 7.822 3.911 3.911 0 0 0 0-7.822zm0 6.356a2.444 2.444 0 1 1 0-4.89 2.444 2.444 0 0 1 0 4.89z"
        />
        <path
          d="M21.396 9.018h-.488a9.142 9.142 0 0 0-.49-1.227l.338-.338c.572-.572.572-1.5 0-2.073l-2.073-2.072a1.467 1.467 0 0 0-2.073 0l-.347.332a9.147 9.147 0 0 0-1.227-.489v-.489c0-.81-.657-1.466-1.467-1.466h-2.933c-.81 0-1.467.656-1.467 1.466v.49c-.42.132-.83.296-1.227.488l-.337-.337a1.467 1.467 0 0 0-2.073 0L3.459 5.376a1.467 1.467 0 0 0 0 2.072l.337.338a9.142 9.142 0 0 0-.488 1.227h-.49c-.81 0-1.466.657-1.466 1.467v2.933c0 .81.657 1.467 1.467 1.467h.489c.133.42.296.83.488 1.227l-.337.337a1.467 1.467 0 0 0 0 2.073l2.073 2.073c.573.572 1.5.572 2.073 0l.347-.328c.397.193.807.356 1.227.49v.488c0 .81.657 1.467 1.467 1.467h2.933c.81 0 1.467-.657 1.467-1.467v-.489c.42-.133.83-.296 1.227-.489l.337.338c.573.571 1.5.571 2.073 0l2.073-2.073c.572-.573.572-1.5 0-2.073l-.337-.338c.192-.396.355-.806.489-1.227h.488c.81 0 1.467-.656 1.467-1.466v-2.934a1.467 1.467 0 0 0-1.467-1.471zm0 4.4h-1.608a7.778 7.778 0 0 1-1.212 2.933l1.134 1.134-2.073 2.073-1.134-1.134a7.773 7.773 0 0 1-2.934 1.212v1.604h-2.933v-1.608a7.773 7.773 0 0 1-2.933-1.213l-1.135 1.134-2.072-2.073 1.134-1.134a7.778 7.778 0 0 1-1.213-2.933H2.82V10.48h1.608A7.778 7.778 0 0 1 5.64 7.546L4.505 6.412l2.073-2.073 1.134 1.134a7.778 7.778 0 0 1 2.934-1.212V2.662h2.933v1.609c1.05.198 2.05.61 2.933 1.212l1.135-1.134 2.073 2.073-1.144 1.13a7.778 7.778 0 0 1 1.212 2.932h1.608v2.934z"
        />
      </g>,
      "height": 20,
      "viewBox": "0 0 24 24",
      "width": 20,
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <path
            d="M12.108 8.04a3.911 3.911 0 1 0 0 7.822 3.911 3.911 0 0 0 0-7.822zm0 6.356a2.444 2.444 0 1 1 0-4.89 2.444 2.444 0 0 1 0 4.89z"
          />,
          <path
            d="M21.396 9.018h-.488a9.142 9.142 0 0 0-.49-1.227l.338-.338c.572-.572.572-1.5 0-2.073l-2.073-2.072a1.467 1.467 0 0 0-2.073 0l-.347.332a9.147 9.147 0 0 0-1.227-.489v-.489c0-.81-.657-1.466-1.467-1.466h-2.933c-.81 0-1.467.656-1.467 1.466v.49c-.42.132-.83.296-1.227.488l-.337-.337a1.467 1.467 0 0 0-2.073 0L3.459 5.376a1.467 1.467 0 0 0 0 2.072l.337.338a9.142 9.142 0 0 0-.488 1.227h-.49c-.81 0-1.466.657-1.466 1.467v2.933c0 .81.657 1.467 1.467 1.467h.489c.133.42.296.83.488 1.227l-.337.337a1.467 1.467 0 0 0 0 2.073l2.073 2.073c.573.572 1.5.572 2.073 0l.347-.328c.397.193.807.356 1.227.49v.488c0 .81.657 1.467 1.467 1.467h2.933c.81 0 1.467-.657 1.467-1.467v-.489c.42-.133.83-.296 1.227-.489l.337.338c.573.571 1.5.571 2.073 0l2.073-2.073c.572-.573.572-1.5 0-2.073l-.337-.338c.192-.396.355-.806.489-1.227h.488c.81 0 1.467-.656 1.467-1.466v-2.934a1.467 1.467 0 0 0-1.467-1.471zm0 4.4h-1.608a7.778 7.778 0 0 1-1.212 2.933l1.134 1.134-2.073 2.073-1.134-1.134a7.773 7.773 0 0 1-2.934 1.212v1.604h-2.933v-1.608a7.773 7.773 0 0 1-2.933-1.213l-1.135 1.134-2.072-2.073 1.134-1.134a7.778 7.778 0 0 1-1.213-2.933H2.82V10.48h1.608A7.778 7.778 0 0 1 5.64 7.546L4.505 6.412l2.073-2.073 1.134 1.134a7.778 7.778 0 0 1 2.934-1.212V2.662h2.933v1.609c1.05.198 2.05.61 2.933 1.212l1.135-1.134 2.073 2.073-1.144 1.13a7.778 7.778 0 0 1 1.212 2.932h1.608v2.934z"
          />,
        ],
        "fill": "black",
        "fillRule": "evenodd",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M12.108 8.04a3.911 3.911 0 1 0 0 7.822 3.911 3.911 0 0 0 0-7.822zm0 6.356a2.444 2.444 0 1 1 0-4.89 2.444 2.444 0 0 1 0 4.89z",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M21.396 9.018h-.488a9.142 9.142 0 0 0-.49-1.227l.338-.338c.572-.572.572-1.5 0-2.073l-2.073-2.072a1.467 1.467 0 0 0-2.073 0l-.347.332a9.147 9.147 0 0 0-1.227-.489v-.489c0-.81-.657-1.466-1.467-1.466h-2.933c-.81 0-1.467.656-1.467 1.466v.49c-.42.132-.83.296-1.227.488l-.337-.337a1.467 1.467 0 0 0-2.073 0L3.459 5.376a1.467 1.467 0 0 0 0 2.072l.337.338a9.142 9.142 0 0 0-.488 1.227h-.49c-.81 0-1.466.657-1.466 1.467v2.933c0 .81.657 1.467 1.467 1.467h.489c.133.42.296.83.488 1.227l-.337.337a1.467 1.467 0 0 0 0 2.073l2.073 2.073c.573.572 1.5.572 2.073 0l.347-.328c.397.193.807.356 1.227.49v.488c0 .81.657 1.467 1.467 1.467h2.933c.81 0 1.467-.657 1.467-1.467v-.489c.42-.133.83-.296 1.227-.489l.337.338c.573.571 1.5.571 2.073 0l2.073-2.073c.572-.573.572-1.5 0-2.073l-.337-.338c.192-.396.355-.806.489-1.227h.488c.81 0 1.467-.656 1.467-1.466v-2.934a1.467 1.467 0 0 0-1.467-1.471zm0 4.4h-1.608a7.778 7.778 0 0 1-1.212 2.933l1.134 1.134-2.073 2.073-1.134-1.134a7.773 7.773 0 0 1-2.934 1.212v1.604h-2.933v-1.608a7.773 7.773 0 0 1-2.933-1.213l-1.135 1.134-2.072-2.073 1.134-1.134a7.778 7.778 0 0 1-1.213-2.933H2.82V10.48h1.608A7.778 7.778 0 0 1 5.64 7.546L4.505 6.412l2.073-2.073 1.134 1.134a7.778 7.778 0 0 1 2.934-1.212V2.662h2.933v1.609c1.05.198 2.05.61 2.933 1.212l1.135-1.134 2.073 2.073-1.144 1.13a7.778 7.778 0 0 1 1.212 2.932h1.608v2.934z",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
      ],
      "type": "g",
    },
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <g
          fill="black"
          fillRule="evenodd"
        >
          <path
            d="M12.108 8.04a3.911 3.911 0 1 0 0 7.822 3.911 3.911 0 0 0 0-7.822zm0 6.356a2.444 2.444 0 1 1 0-4.89 2.444 2.444 0 0 1 0 4.89z"
          />
          <path
            d="M21.396 9.018h-.488a9.142 9.142 0 0 0-.49-1.227l.338-.338c.572-.572.572-1.5 0-2.073l-2.073-2.072a1.467 1.467 0 0 0-2.073 0l-.347.332a9.147 9.147 0 0 0-1.227-.489v-.489c0-.81-.657-1.466-1.467-1.466h-2.933c-.81 0-1.467.656-1.467 1.466v.49c-.42.132-.83.296-1.227.488l-.337-.337a1.467 1.467 0 0 0-2.073 0L3.459 5.376a1.467 1.467 0 0 0 0 2.072l.337.338a9.142 9.142 0 0 0-.488 1.227h-.49c-.81 0-1.466.657-1.466 1.467v2.933c0 .81.657 1.467 1.467 1.467h.489c.133.42.296.83.488 1.227l-.337.337a1.467 1.467 0 0 0 0 2.073l2.073 2.073c.573.572 1.5.572 2.073 0l.347-.328c.397.193.807.356 1.227.49v.488c0 .81.657 1.467 1.467 1.467h2.933c.81 0 1.467-.657 1.467-1.467v-.489c.42-.133.83-.296 1.227-.489l.337.338c.573.571 1.5.571 2.073 0l2.073-2.073c.572-.573.572-1.5 0-2.073l-.337-.338c.192-.396.355-.806.489-1.227h.488c.81 0 1.467-.656 1.467-1.466v-2.934a1.467 1.467 0 0 0-1.467-1.471zm0 4.4h-1.608a7.778 7.778 0 0 1-1.212 2.933l1.134 1.134-2.073 2.073-1.134-1.134a7.773 7.773 0 0 1-2.934 1.212v1.604h-2.933v-1.608a7.773 7.773 0 0 1-2.933-1.213l-1.135 1.134-2.072-2.073 1.134-1.134a7.778 7.778 0 0 1-1.213-2.933H2.82V10.48h1.608A7.778 7.778 0 0 1 5.64 7.546L4.505 6.412l2.073-2.073 1.134 1.134a7.778 7.778 0 0 1 2.934-1.212V2.662h2.933v1.609c1.05.198 2.05.61 2.933 1.212l1.135-1.134 2.073 2.073-1.144 1.13a7.778 7.778 0 0 1 1.212 2.932h1.608v2.934z"
          />
        </g>,
        "height": 20,
        "viewBox": "0 0 24 24",
        "width": 20,
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <path
              d="M12.108 8.04a3.911 3.911 0 1 0 0 7.822 3.911 3.911 0 0 0 0-7.822zm0 6.356a2.444 2.444 0 1 1 0-4.89 2.444 2.444 0 0 1 0 4.89z"
            />,
            <path
              d="M21.396 9.018h-.488a9.142 9.142 0 0 0-.49-1.227l.338-.338c.572-.572.572-1.5 0-2.073l-2.073-2.072a1.467 1.467 0 0 0-2.073 0l-.347.332a9.147 9.147 0 0 0-1.227-.489v-.489c0-.81-.657-1.466-1.467-1.466h-2.933c-.81 0-1.467.656-1.467 1.466v.49c-.42.132-.83.296-1.227.488l-.337-.337a1.467 1.467 0 0 0-2.073 0L3.459 5.376a1.467 1.467 0 0 0 0 2.072l.337.338a9.142 9.142 0 0 0-.488 1.227h-.49c-.81 0-1.466.657-1.466 1.467v2.933c0 .81.657 1.467 1.467 1.467h.489c.133.42.296.83.488 1.227l-.337.337a1.467 1.467 0 0 0 0 2.073l2.073 2.073c.573.572 1.5.572 2.073 0l.347-.328c.397.193.807.356 1.227.49v.488c0 .81.657 1.467 1.467 1.467h2.933c.81 0 1.467-.657 1.467-1.467v-.489c.42-.133.83-.296 1.227-.489l.337.338c.573.571 1.5.571 2.073 0l2.073-2.073c.572-.573.572-1.5 0-2.073l-.337-.338c.192-.396.355-.806.489-1.227h.488c.81 0 1.467-.656 1.467-1.466v-2.934a1.467 1.467 0 0 0-1.467-1.471zm0 4.4h-1.608a7.778 7.778 0 0 1-1.212 2.933l1.134 1.134-2.073 2.073-1.134-1.134a7.773 7.773 0 0 1-2.934 1.212v1.604h-2.933v-1.608a7.773 7.773 0 0 1-2.933-1.213l-1.135 1.134-2.072-2.073 1.134-1.134a7.778 7.778 0 0 1-1.213-2.933H2.82V10.48h1.608A7.778 7.778 0 0 1 5.64 7.546L4.505 6.412l2.073-2.073 1.134 1.134a7.778 7.778 0 0 1 2.934-1.212V2.662h2.933v1.609c1.05.198 2.05.61 2.933 1.212l1.135-1.134 2.073 2.073-1.144 1.13a7.778 7.778 0 0 1 1.212 2.932h1.608v2.934z"
            />,
          ],
          "fill": "black",
          "fillRule": "evenodd",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M12.108 8.04a3.911 3.911 0 1 0 0 7.822 3.911 3.911 0 0 0 0-7.822zm0 6.356a2.444 2.444 0 1 1 0-4.89 2.444 2.444 0 0 1 0 4.89z",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M21.396 9.018h-.488a9.142 9.142 0 0 0-.49-1.227l.338-.338c.572-.572.572-1.5 0-2.073l-2.073-2.072a1.467 1.467 0 0 0-2.073 0l-.347.332a9.147 9.147 0 0 0-1.227-.489v-.489c0-.81-.657-1.466-1.467-1.466h-2.933c-.81 0-1.467.656-1.467 1.466v.49c-.42.132-.83.296-1.227.488l-.337-.337a1.467 1.467 0 0 0-2.073 0L3.459 5.376a1.467 1.467 0 0 0 0 2.072l.337.338a9.142 9.142 0 0 0-.488 1.227h-.49c-.81 0-1.466.657-1.466 1.467v2.933c0 .81.657 1.467 1.467 1.467h.489c.133.42.296.83.488 1.227l-.337.337a1.467 1.467 0 0 0 0 2.073l2.073 2.073c.573.572 1.5.572 2.073 0l.347-.328c.397.193.807.356 1.227.49v.488c0 .81.657 1.467 1.467 1.467h2.933c.81 0 1.467-.657 1.467-1.467v-.489c.42-.133.83-.296 1.227-.489l.337.338c.573.571 1.5.571 2.073 0l2.073-2.073c.572-.573.572-1.5 0-2.073l-.337-.338c.192-.396.355-.806.489-1.227h.488c.81 0 1.467-.656 1.467-1.466v-2.934a1.467 1.467 0 0 0-1.467-1.471zm0 4.4h-1.608a7.778 7.778 0 0 1-1.212 2.933l1.134 1.134-2.073 2.073-1.134-1.134a7.773 7.773 0 0 1-2.934 1.212v1.604h-2.933v-1.608a7.773 7.773 0 0 1-2.933-1.213l-1.135 1.134-2.072-2.073 1.134-1.134a7.778 7.778 0 0 1-1.213-2.933H2.82V10.48h1.608A7.778 7.778 0 0 1 5.64 7.546L4.505 6.412l2.073-2.073 1.134 1.134a7.778 7.778 0 0 1 2.934-1.212V2.662h2.933v1.609c1.05.198 2.05.61 2.933 1.212l1.135-1.134 2.073 2.073-1.144 1.13a7.778 7.778 0 0 1 1.212 2.932h1.608v2.934z",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
        ],
        "type": "g",
      },
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
